portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Loads the comments on every page without having to redundantly specify in HTML.
+window.onload = function() {
+  getCommentsFromServlet();
```

Comment:
I feel like in addition to loading the existing database comments, you could also refactor the "new comment form" html into the onload function so you don't have to repeat it on each page.

Reply:
It took me forever but this is finally done! It is also hideous, unfortunately. Normally you'd use JQuery or PHP to do something like this.